src/main/java/aquality/appium/mobile/elements/actions/SwipeActions.java

```diff
@@ -0,0 +1,102 @@
+package aquality.appium.mobile.elements.actions;
+
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.configuration.ISwipeConfiguration;
+import aquality.appium.mobile.elements.ScreenSizeType;
+import aquality.appium.mobile.elements.SwipeDirection;
+import aquality.appium.mobile.elements.interfaces.IElement;
+import aquality.selenium.core.logging.Logger;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.offset.PointOption;
+import org.openqa.selenium.Dimension;
+
+import static io.appium.java_client.touch.WaitOptions.waitOptions;
+
+public class SwipeActions {
```

Comment:
I would like to make this class non-static, with an interface and two different implementations: for an element(cause we can start swiping from the element) and for the application(cause we can ). Please take a look at example here: 
https://github.com/aquality-automation/aquality-winappdriver-dotnet/blob/master/Aquality.WinAppDriver/src/Aquality.WinAppDriver/Actions/IMouseActions.cs
https://github.com/aquality-automation/aquality-winappdriver-dotnet/blob/master/Aquality.WinAppDriver/src/Aquality.WinAppDriver/Actions/MouseActions.cs
https://github.com/aquality-automation/aquality-winappdriver-dotnet/blob/master/Aquality.WinAppDriver/src/Aquality.WinAppDriver/Elements/Actions/MouseActions.cs

Is it possible?

Also, scroll doesn't sound like a SwipeAction, so we probably should name this class "TouchActions", what do you think about it?